src/Client.js

```diff
@@ -1096,20 +1096,20 @@ class Client extends EventEmitter {
     }
 
     /**
-     * Returns the contact ID's profile picture URL, if privacy settings allow it
+     * Returns the chat ID's profile picture URL, if the client can view it
      * @param {string} contactId the whatsapp user's ID
```

Comment:
```suggestion
```
If you decided to rename the `contactId` parameter to `chatId` I think it would be better to remove this line